advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx

```diff
@@ -0,0 +1,309 @@
+2. Click `New` option.
+
+3. Right click `Hosts, Servers, Tables` and select `New Host`.  
+
+4. Enter information for the `Host`, `Host` is the machine where Connect CDC Director is running.
+
+![Create New Host](Images/newhost.png)
+
+5. Right Click on the newly added `Host` and select `Test Connection`, a success message will be displayed.
+
+6. Right Click on the newly added `Host` and select `New Server`.
+
+7. Enter Connection Information for EDB Postgres Advanced Server as `Source` Instance (Server name is where the database name is given).
+
+![Create New Server](Images/newserver.png)
+
+8. Right Click on the newly added `Server` and select `Install Source Metabase`.
+
+9. Provide EDB Postgres Advanced Server Source Instance User and Password on `User ID and Password` Screen.
+
+10. Provide password of `rpuser` (refer to the user name and password given on `New Server` Screen) when prompted on the User ID and Password screen.
+
+!!! Note
+     `rpuser` is the replication user added by default by Precisely Connect CDC.
+
+11. Click Ok on `Install Source Metabase` screen.
+
+12. A success message will be displayed once the `Source Metabase` is created on the EDB Postgres Advanced Server `Source` Instance.
+
+13. Open pgAdmin and connect to your EDB Postgres Advanced Server Source instance, navigate to Schemas and under the Schemas `rpuser` Schema has been created.
+
+![pgAdmin](Images/pgadming.png)
+
+14. Navigate back to Connect CDC Director and right Click on the newly added Server and select `Test Connection`. A success message will be displayed.
+
+15. Right Click on the newly added `Server` and select `Prepare User Database`, this will add the replication user to the public database.
+
+16. On `Database/Schema name` Screen provide the schema name for the schema you wish to replicate. In our example we are using schema `public`.
+
+17. On the `User ID and Password` Screen, provide EDB Postgres Advanced Server Source Instance `User` and `Password`.
+
+18. Click Ok on `Prepare User Database` Screen.
+
+19. A success message will be displayed once the `Prepare User Database` operation is successful.
+
+20. Right Click on the newly added `Server` and select `Refresh Available Tables`.
+
+21. Click on the Refresh list and then select schema `public` and click ok.
+
+![ Refresh Available Tables](Images/refreshtables.png)
+
+22. To display the available tables click newly added `Server` —> `Tables` —> `Available Tables` —> `Public` (Schema name), a list of available tables will be displayed.
+
+![Available Tables](Images/availabletables.png)
+
+## Configure Postgres Distribution as Target Instance in Connect CDC Director
+
+The following steps demonstrate the setup and configuration of EDB Postgres Advanced Server as target instance to Connect CDC Director. 
+
+1. Right click `DBMS Servers` and `New Server`.
+
+2. Provide Connection Information for the EDB Postgres Advanced Server Target Instance.
+
+![Target Server](Images/targetserver.png)
```

Comment:
Could we also crop this a bit more on the bottom?

Reply:
done